src/components/consul/detail/ConsulCommentListItem.js

```diff
@@ -36,14 +36,14 @@ const deleteCommentAlert = (onDelete) =>
 
 /* eslint-disable complexity */
 /* NOTE: we need to check a lot for presence, so this is that complex */
-export const ConsulCommentListItem = ({ commentItem, onRefresh, replyList, navigation }) => {
-  const [showResponse, setShowResponse] = useState(false);
-  const [showReply, setShowReply] = useState(false);
+export const ConsulCommentListItem = ({ commentItem, refetch, replyList, navigation }) => {
   const [isLoading, setIsLoading] = useState(false);
+  const [isShowReply, setIsShowReply] = useState(false);
+  const [isShowResponse, setIsShowResponse] = useState(false);
 
   const { control, handleSubmit, reset } = useForm({
     defaultValues: {
-      comment: ''
+      comment: null
```

Comment:
why was this changed to null?

Reply:
i thought `null` was more accurate both in terms of readability and in terms of disk space.

Reply:
does the react-hook-form docs say something about that?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/521#discussion_r862853106